### DISCUSSION TITLE
import image ga work

### DISCUSSION THREAD

**Discussion by DERMOIIN** [deaafrizal/forum#54] [Troubleshooting Room] [answered]
![Screenshot (448)](https://github.com/deaaprizal/forum/assets/110816806/7514db65-f72d-4fe9-9623-756f1362741a)
Ini saya udah masukkin file gambar nya ke direktori public, setelahnya saya ingin masukkan gambar ini ke Hero.js dengan fungsi import, tapi entah kenapa gabisa. Saya pake router dan src/, tolong bantu saya yang bisa.

**Comment by Mikael2909** [comment]
Coba pakai import image dari next navigation, di konten bang Dea yang buat website anime ada tuh cara nya
Pakai nya mudah kok

**Comment by ikhlasdansantai** [comment, marked as answer]
klo disimpen di folder public, saya biasanya pake cara ini

```jsx
import iniGambar from "/public/logo.png"
```

**Reply by DERMOIIN** [reply]
thanks bang
